src/test/kotlin/RacingCarTest.kt

```diff
@@ -0,0 +1,82 @@
+package racingCar
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class RacingCarTest {
+
+    lateinit var car: Car
```

Comment:
질문 남겨주셔서 확인해봤는데

이런 경우라면 아래와 같이 사용해도 괜찮을 것 같네요

각 테스트별 Car는 격리되어 값을 변경해도 영향받지 않습니다.

```suggestion
    private val car = Car()
```